crates/lib3h/src/transport/memory_mock/ghost_transport_memory.rs

```diff
@@ -411,30 +410,30 @@ mod tests {
             Some(bound_transport3_address.clone())
         );
 
-        // check that machine_ids were advertised
+        // check that transport_ids were advertised
```

Comment:
Here's a place where we use the term transport_id and the code is looking for a `mem://`  Maybe the comment is the error?